CONTRIBUTING.md

```diff
@@ -9,6 +9,25 @@ refactoring, additional comments, and so forth. Let's try to raise the
 bar, and clean things up as we go. Try to leave code in a better shape
 than it was before.
 
+## Pre-commit hook
+
+We have a sample pre-commit hook in `pre-commit.py`.
+To set it up, run:
+
+```bash
+ln -s ../../pre-commit.py .git/hooks/pre-commit
+```
+
+This will run following checks on staged files before each commit:
+- `cargo fmt`
```

Comment:
It's not exactly `cargo fmt`, it's `rustfmt`, so it does not fix codestyle errors, but checks it (I think, it's important t mention).

Reply:
> Also, it would be nice to have a flag to apply formatting automatically (if it's possible).

We've got `./scripts/reformat` for this